demos/memes-elastic/README.md

```diff
@@ -1,7 +1,24 @@
-# Memes Elastic Demo
+# Elastic Demo: Analyzing Internet News
 
 # Setup
 
+This demo loads a data set of Internet news snippets from April 2009, collected by Memetracker (see [data source](#data-source)), into ElasticSearch, then uses Juttle with elastic adapter to read this data, search the news, compute aggregations, and visualize in the browser. The demo uses docker containers and maps local Juttle programs to be run inside the `outrigger` container.
+
+## Prerequisites
+
+Run docker machine:
+
+```
+docker-machine start default
```

Comment:
docker-machine does not exist on my linux setup so I'm not sure writing things using this script is the best way as this would be a mac only script ? (not sure where that comes from).